src/osd/PeeringState.cc

```diff
@@ -1461,7 +1472,6 @@ map<pg_shard_t, pg_info_t>::const_iterator PeeringState::find_best_info(
   /* See doc/dev/osd_internals/last_epoch_started.rst before attempting
    * to make changes to this process.  Also, make sure to update it
    * when you find bugs! */
-  eversion_t min_last_update_acceptable = eversion_t::max();
```

Comment:
Why?  Is this just to delay declaration?

Reply:
Bluntly, this code gets looked at every time there's an emergency with peering.  If you are going to make minor cosmetic changes, you need to be more specific in the commit message as to why so that future readers can comfortably skip your commit.

Reply:
Yes, the sole purpose of the change was to move the declaration closer to its usage. Commented that in the commit description.